apps/server/src/modules/oauth/controller/oauth-sso.controller.spec.ts

```diff
@@ -85,7 +85,7 @@ describe('OAuthController', () => {
 	});
 
 	describe('requestAuthToken', () => {
-		const currentUser: ICurrentUser = { userId: 'userId' } as ICurrentUser;
+		const currentUser = iCurrentUserFactory.buildWithId({ userId: 'userId' });
 		const oauthClientId = 'clientId';
```

Comment:
I think `build` is enough as `ICurrentUser` does not have an id.
```suggestion
		const currentUser = iCurrentUserFactory.build({ userId: 'userId' });
```